index.html

```diff
@@ -0,0 +1,194 @@
+          </div>
+        </div>
+      
+      </div>
+    </div>
+    </section>
+    <section class="my-5" id="product-card">
+          <div class="row">
+            <div class="col-md-4 d-flex justify-content-around">
+              <div class="card text-center">
+                <div class="card-header">
+                  <img class="img-responsive" height="200" src="https://cdn.pixabay.com/photo/2017/08/17/08/20/online-shopping-2650383_960_720.jpg" alt="product-image">
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">$600</h3>
+                    <p>
+                      <ul>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                      </ul>
+                    </p>
+                  <a href="#" class="btn btn-primary">Go somewhere</a>
+                </div>
+                <div class="card-footer text-muted">
+                  2 days ago
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4 d-flex justify-content-around">
+              <div class="card text-center">
+                <div class="card-header">
+                  <img class="img-responsive" height="200" src="https://cdn.pixabay.com/photo/2017/08/17/08/20/online-shopping-2650383_960_720.jpg" alt="product-image">
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">$900</h3>
+                  <p>
+                    <ul>
+                      <li>Lorem ipsum dolor sit amet consectetur</li>
+                      <li>Lorem ipsum dolor sit amet consectetur</li>
+                      <li>Lorem ipsum dolor sit amet consectetur</li>
+                      <li>Lorem ipsum dolor sit amet consectetur</li>
+                    </ul>
+                  </p>
+                  <a href="#" class="btn btn-primary">Go somewhere</a>
+                </div>
+                <div class="card-footer text-muted">
+                  2 days ago
+                </div>
+              </div>
+            </div>
+             
+            <div class="col-md-4 d-flex justify-content-around">
+              <div class="card text-center">
+                <div class="card-header">
+                  <img class="img-responsive" height="200" src="https://cdn.pixabay.com/photo/2017/08/17/08/20/online-shopping-2650383_960_720.jpg" alt="product-image">
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">$1,200</h3>
+                  <p>
```

Comment:
please use div tag instead of p.

Reply:
- [ ] Good useof the inline comment.👏 However, make use that when you are requesting a change to explain `why` you requested the change. Here what do you think instead of just saying `please use div tag instead of p` but you explain why you request the change. Doing so is going the help the project author not repeat the same mistake next time.👌
You can say something like `Please because of  semantic reasons, I would recommend you using a div tag instead of a p tag` and to help him/her better you can provide a link to more learn about sementic in html.
### Here is an example of a good comment 👇️
` The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?`

- [ ] I think it would be better if you have at least two comments. This comment up 👆️ here you can edit it to meet the requirement. 
- [ ] If you don't see anything else to request a changem you can intentionnaly add one for instance leave  a commented code and ask for a change while explaining why we should not have commented code on our codebase.